spec/spec.md

```diff
@@ -87,88 +86,66 @@ select was the
 how to format a request that verifiable information be presented. The
 [Presentation Exchange specification](https://identity.foundation/presentation-exchange/spec/v1.0.0/)
 was recently published by the Decentralized Identity Foundation. It defines a
-`Presentation Definition` data object which may be used by a relying party to
-request information, and a `Presentation Submission` object which describes the
-relationship between the `Presentation Definition` and the submitted verifiable
-information.
-
-Similarly, in issuance situations the credentials on offer by a given issuer need to be defined in objects that can be passed back and forth. In these flows, a "sister specification" that spun out of the Presentation Exchange work-item group defines those objects: [Credential Manifest](https://identity.foundation/credential-manifest/#credential-manifest-2), also a work item of the DIF Claims and Credentials WG.
+`Presentation Definition` data object which may be used by a verifier (i.e., a 
+"relying party" in OIDC parlance) to request information, and a `Presentation 
+Submission` object which describes the relationship between the `Presentation 
+Definition` and the submitted verifiable information.
 
 Since
 [Presentation Exchange](https://identity.foundation/presentation-exchange/spec/v1.0.0/)
 supports the use of `Verifiable Presentations` as a response to a `Presentation
 Definition`, while remaining agnostic to transport protocols, it is ideal for
 our purposes.
 
-### Communication, Transport, and Protocol
-Previously described components were, for the most part, agreed upon readily.
-The bulk of the conversations at IIW centered around communication and protocol
-options and sought to answer the following questions:
-- "How should the data objects be communicated from one party to another?"
-- "How ought they to be transported securely?"
-- "What protocol could be used for the exchange?"
-
-#### DIDComm
-Before participants settled on using elements of
+### Protocol Considerations
+Before fixing the scope of this work item and settling on using elements of
 [DIDComm v2.0](https://identity.foundation/didcomm-messaging/spec/) to securely
 communicate the data objects, there was a long and lively discussion about other
-options. 
+options at IIW. 
 
-[CHAPI](https://w3c-ccg.github.io/credential-handler-api/) is an API for
+- [CHAPI](https://w3c-ccg.github.io/credential-handler-api/) is an API for
 exchanging data between websites and browsers. It was seen as too limited to a
 particular technology to be widely useful for wallet applications on smart
 phones and other devices without significant changes.
 
-[VC-HTTP-API](https://w3c-ccg.github.io/vc-http-api/) was seen as promising, but
-in its current state lacks an API for a `Verifiable Credential` holder. A number
-of participants expressed a desire for the WACI specification to strive to be
+- [VC-API](https://w3c-ccg.github.io/vc-api/) (fka VC-HTTP-API) was seen as promising, 
```

Comment:
I will wait for this to be merged before taking a swing at this: https://github.com/decentralized-identity/waci-didcomm/issues/14

Reply:
don't hurt me